Example/Podfile

```diff
@@ -1,138 +1,136 @@
+#remove next two lines after Firebase 4.6.0 release and before merge
+source 'sso://cpdc-internal/spec'
+source 'https://github.com/CocoaPods/Specs.git'
 
 use_frameworks!
 
+pod 'FirebaseCore', :path => '../'
+
 target 'Core_Example_iOS' do
   platform :ios, '8.0'
-
-  pod 'FirebaseCommunity/Core', :path => '../'
+  pod 'Firebase/Core', '4.6.0'
```

Comment:
It makes me deeply uncomfortable that you're mixing `'FirebaseCore'` and `'Firebase/Core'` in a single Podfile. It means that there are multiple sources of the core headers which could lead to highly unexpected results.

This seems like something we should avoid at all costs.

Reply:
They go to the same place. Firebase/Core depends on FirebaseCore.  While we're running parallel builds, this line serves as a way to keep the open source repo aligned with the official pushed version of Firebase. I added a comment.

Reply:
I don't understand "the forcing function for the Firebase pod". What are we forcing?

It seems like `Example` has a dependency on `Firebase/Core`, but that we're overriding the path of `FirebaseCore`, no?